script/upgrades/MU01/governance/MU01-CGP-Phase2.sol

```diff
@@ -238,6 +256,141 @@ contract MU01_CGP_Phase2 is ICGPBuilder, GovernanceScript {
     }
   }
 
+  function proposal_configureBreakerBox() public {
+    // Add the Median Delta Breaker to the breaker box with the trading mode '1' -> No Trading
+    if (breakerBox != address(0) || BreakerBox(breakerBox).breakerTradingMode(medianDeltaBreaker) == 0) {
+      transactions.push(
+        ICeloGovernance.Transaction(
+          0,
+          breakerBox,
+          abi.encodeWithSelector(BreakerBox(0).addBreaker.selector, medianDeltaBreaker, 1)
+        )
+      );
+    }
+
+    // Add the Value Delta Breaker to the breaker box with the trading mode '2' -> No Trading
+    if (breakerBox != address(0) || BreakerBox(breakerBox).breakerTradingMode(valueDeltaBreaker) == 0) {
+      transactions.push(
+        ICeloGovernance.Transaction(
+          0,
+          breakerBox,
+          abi.encodeWithSelector(BreakerBox(0).addBreaker.selector, valueDeltaBreaker, 2)
+        )
+      );
+    }
+
+    // Set rate feed dependencies cEUR/USDC, cBRL/USDC for cUSD/USDC rate feed
+    transactions.push(
+      ICeloGovernance.Transaction(
+        0,
+        breakerBox,
+        abi.encodeWithSelector(
+          BreakerBox(0).setRateFeedDependencies.selector,
+          cUSDUSDCConfig.referenceRateFeedID,
+          cEURUSDCConfig.referenceRateFeedID,
+          cBRLUSDCConfig.referenceRateFeedID
+        )
+      )
+    );
+
+    // Enable Median Delta Breaker for rate feeds
+    for (uint256 i = 0; i < poolConfigs.length; i++) {
+      if (poolConfigs[i].isMedianDeltaBreakerEnabled) {
+        transactions.push(
+          ICeloGovernance.Transaction(
+            0,
+            breakerBox,
+            abi.encodeWithSelector(
+              BreakerBox(0).toggleBreaker.selector,
+              medianDeltaBreaker,
+              poolConfigs[i].referenceRateFeedID,
+              true
+            )
+          )
+        );
+      }
+    }
+
+    // Enable Value Delta Breaker for rate feeds
```

Comment:
I am not sure whether we want the value delta breaker for all rateFeeds in phase 1 we only enabled it for cUSD/USDC